client/request.go

```diff
@@ -104,8 +104,14 @@ func (client *APIClient) hijack(ctx context.Context, path string, query url.Valu
 	}
 
 	if tcpConn, ok := conn.(*net.TCPConn); ok {
-		tcpConn.SetKeepAlive(true)
-		tcpConn.SetKeepAlivePeriod(30 * time.Second)
+		err = tcpConn.SetKeepAlive(true)
```

Comment:
I hope we can use the format of 
``` Go
if err := tcpConn.SetKeepAlive(); err != nil {
    return ...
}
```
Please address these all

Reply:
@yeya24 
thanks for your suggestion.I changed it.